hack/crossplane/azure-crossplane.yaml

```diff
@@ -133,7 +133,7 @@ typeTransformers:
     property: Copy
     ifType:
       group: deploymenttemplate
-      version: v1alpha1api20190401
+      version: "2019-04-01"
```

Comment:
minor: Can this just be `2019-04-01`? Does it need the quotes?

Reply:
Needs the quotes, else it seems to get interpreted as a date.